server/src/controllers/menuControllers.js

```diff
@@ -8,6 +8,32 @@ class MenuControllers {
       data: menu
     });
   }
+
+  // Post Menu for the single day
+  static postMenu(req, res) {
+    if (!req.body.name) {
```

Comment:
Basically, validation should be done outside your controllers' `CRUD`  methods. Class methods concerned with crud operations should do nothing else besides crud operations. An operation like validation should be abstracted into a middleware class/fn, so that only valid `queries` end up in your controller methods. It makes your controller lean and debugging easier. If there's a failure in any part of the system, you are able to isolate it in no time.

```
ex.

validationFn = (req, res, next) => {
  // perform validation
  if (req is valid) {
    return next(req);
  }

  // req is not valid
  res.status(/* whatever */).send(errMsg);
}

menuRoutes.post('/menu', validationFn, handler);
```